tests/test_datasets.py

```diff
@@ -14,28 +20,60 @@
 # We should NOT care what the stores actually do - that's
 # what the /store tests are for, so we mock a store.
 
+endpoint_url = "/datasets"
```

Comment:
your setting this as a constant value and constants are upper case in python, so ENDPOINT_URL =